aggregator.go

```diff
@@ -108,6 +115,27 @@ func startService() {
 	os.Exit(ExitStatusOK)
 }
 
+func waitForServiceToStart() {
+	for {
+		if serverInstance != nil && consumerInstance != nil {
+			// everything was initialized
+			break
+		}
+		time.Sleep(500 * time.Millisecond)
+	}
+}
+
+func stopService() {
+	err := serverInstance.Stop(context.TODO())
+	if err != nil {
+		log.Fatal(err)
```

Comment:
hmm in this case it might be probably more correct to try to Close() consumer before log.Fatal. WDYT?

Reply:
fixed? https://github.com/RedHatInsights/insights-results-aggregator/pull/99/commits/712348e379fccd08bf4e6067417da165a09bc5fe